testplan/exporters/testing/pdf/renderers/base.py

```diff
@@ -8,12 +8,9 @@
 
 from reportlab.platypus import Paragraph
 
-try:
-    from html import escape as _orig_escape
+from html import escape as _orig_escape
 
-    escape = functools.partial(_orig_escape, quote=False)
-except ImportError:
-    from cgi import escape
+escape = functools.partial(_orig_escape, quote=False)
```

Comment:
This is just used in a single place, no need to overwrite it with the partial.

Reply:
fixed